spec/notifications-status-view-spec.js

```diff
@@ -0,0 +1,84 @@
+/** @babel */
```

Comment:
Don't use Babel.  Pretty sure this just entails changing the imports to requires.